python_scripts/03_categorical_pipeline.py

```diff
@@ -93,9 +93,10 @@
 categorical_columns
 
 # %% [markdown]
-# We can create the selector by passing the data type to include or exclude.
-# Once the selector created, we can pass the input dataset and the name of
-# the selected columns will be returned.
+# Here, we created the selector by passing the dtype to include ;
```

Comment:
Honestly I am not a super big fan of `;` we are not writing a novel ...

Only my opinion but shorter sentences and `.` make it easier to follow.

Reply:
Ignore this comment for now, I needed to rant a little bit ...